app/controllers/account_activations_controller.rb

```diff
@@ -0,0 +1,14 @@
+class AccountActivationsController < ApplicationController
+  def edit
+    user = User.find_by(email: params[:email])
+    if user && !user.activated? && user.authenticated?(:activation, params[:id])
```

Comment:
We could clean this up a bit, maybe create a method called `can_activate_user?` and move that if condition to newly created function.

```suggestion
    if can_activate_user?
    
    def can_activate_user? 
      user && !user.activated? && user.authenticated?(:activation, params[:id])
    end
```

we can also move the logic inside the if block to a separate method and call it `activate_and_redirect_user`

Reply:
good suggestion @LindoDev , though I'm happy to keep the function body since it's small & readable :)